types/mdx/preact-tests.tsx

```diff
@@ -370,29 +314,29 @@ const Div = customComponents.div!;
     }}
 />;
 
-// $ExpectType TestElementType
+// $ExpectType Element
 <MyMKDNComponent />;
 
-// $ExpectType TestElementType
+// $ExpectType Element
 <MyMKDNComponent
     the="answer"
     to={{ life: "the" }}
     universe={() => "and"}
     everything={42}
     components={{
         a(props) {
-            // $ExpectType AnchorProps
+            // $ExpectType HTMLAttributes<HTMLAnchorElement>
             props;
             return null;
         },
         div(props) {
-            // $ExpectType DivProps
+            // $ExpectType HTMLAttributes<HTMLDivElement>
             props;
             return <div {...props} />;
         },
         img: CustomImageComponent,
         // @ts-expect-error
-        video: CustomImageComponent,
+        input: CustomImageComponent,
```

Comment:
is there something about input that makes it a better test than `video`?

Reply:
Yes, apparently input props are assignable to image props in Preact.